test/e2e-olm/utils.go

```diff
@@ -366,3 +350,38 @@ func getCertificateContents(name, uuid string) []byte {
 	filename := fmt.Sprintf("%s/%s/%s", exampleSecretsPath, uuid, name)
 	return utils.GetFileContents(filename)
 }
+
+func getProjectRootPath(projectRootDir string) string {
+	fmt.Println("### getProjectRootPath is running")
+	cwd, err := os.Getwd()
+	cwdOrig := cwd
+	if err != nil {
+		panic(err)
+	}
+	for {
+		if strings.HasSuffix(cwd, "/"+projectRootDir) {
+			return cwd
+		}
+		lastSlashIndex := strings.LastIndex(cwd, "/")
+		if lastSlashIndex == -1 {
+			panic(cwdOrig + " did not contain /" + projectRootDir)
+		}
+		cwd = cwd[0:lastSlashIndex]
+		fmt.Printf("cwd %v\n", cwd)
+	}
+}
+
+func waitForDeleteObject(t *testing.T, client client.Client, key types.NamespacedName, obj runtime.Object, retryInterval, timout time.Duration) error {
+	return wait.Poll(retryInterval, timeout, func() (done bool, err error) {
+		k8sClient.Get(context.Background(), key, obj)
```

Comment:
No error checking for `Get`, missing?

Reply:
Thanks and will fix this.

Reply:
@periklis , after a second thought, I think there is no need to check error returned from `Get`. This is because the next step `Delete` will handle both cases: if error=nil, `Delete` will try to delete the object. If error != nil, Delete will return false and try repeatedly. WDYT?

Reply:
SQTM